### DISCUSSION TITLE
Can't deploy code in lesson 3...

### DISCUSSION THREAD

**Discussion by pythoulon** [smartcontractkit/full-blockchain-solidity-course-py#29] [Q&A] [answered]
When I try to deploy the code, I get a message regarding the contract possibly being abstract, or missing an implementation...

![image](https://user-images.githubusercontent.com/11643101/133638506-e1a8b14f-b0fc-4144-ba2a-c70f282964dd.png)

I must be doing something wrong, but I can't figure it out. I just copied the code from Github; I'm on the Rinkeby test network, and I have 18,75 ETH in my Metamask wallet there...
Any idea ? Thx.

**Comment by rlfleming93** [comment, marked as answer]
Check the contract you are deploying, looks like youre trying to deploy the chainlink imported contract not your FundMe.sol
![Capture](https://user-images.githubusercontent.com/16881025/133639249-7213ed2d-87c0-4a7a-94c9-91239b5c762e.PNG)

**Reply by pythoulon** [reply]
Yes of course... sorry about this!

**Reply by PatrickAlphaC** [reply]
@pythoulon no worries! Thank you for asking this question!

@rlfleming93 thank you for the answer!
